libraries/AP_ADSB/AP_ADSB_uAvionix_UCP.cpp

```diff
@@ -87,12 +87,12 @@ void AP_ADSB_uAvionix_UCP::update()
         }
     } // while nbytes
 
-   if (now_ms - run_state.last_packet_Transponder_Control_ms >= 1000) {
+   if (now_ms - always_realtime_interval_ms(run_state.last_packet_Transponder_Control_ms) >= 1000) {
```

Comment:
make it REALTIME_INTERVAL_MS() macro, macro is pass-thru on real hw, takes a subsystem ID
add a SIM_REALHW_MASK bitmask